dessia_common/tests.py

```diff
@@ -348,7 +348,7 @@ def __init__(self, p_1: float, p_2: float, p_3: float, p_4: float, p_5: float, n
 
     @property
     def test_prop(self):
-        """Factice property for some tests."""
+        """Dummy property for some tests."""
```

Comment:
If you get a conflict where to other where "Fake" is proposed, please keep "Dummy", as you proposed. It is way better, indeed